amida-tech/indaba-client
# update react and stylelint versions, fix associated styles

#### What does this PR do?
Corrects peer dependency warnings.

#### Related JIRA tickets:
INBA-588

#### How should this be manually tested?
Run `yarn` in a fresh repo (no `node_modules`) and ensure no warnings other than the remaining two that are expected.

#### Background/Context
Note that there are still two peer dep issues. One of these is a
permanent issue with stylelint 7 versus 8, so I opted for the latest
version. One package requires “*” for prop-types, which is dumb, so we
are ignoring it.
